File: 35-search-insert-position/search-insert-position.php
class Solution {

    /**
     * @param Integer[] $nums
     * @param Integer $target
     * @return Integer
     */
    function searchInsert($nums, $target) {
        $help=0;

        $i=0;
        while($i<=count($nums)-1){
            if($nums[$i] == $target){
                return $i;
            }
            $i++;
        }

        for($i=0; $i<=count($nums)-1; $i++){
            if($target > $nums[$i] && $target <$nums[$i +1]){
                return $i +1;
            }
        }
        
         for($i=0; $i<=count($nums)-1; $i++){
            if($target < $nums[$i] ){
              $help++;
            }
        }
        

        if($help == count($nums)){
            return 0;
        }

        return count($nums);
     
        
    }
}

File: 35-search-insert-position/search-insert-position.py
class Solution(object):
    def searchInsert(self, nums, target):
     nums.sort()
     len_ =len(nums)
     for i in range(len_):
        if nums[i] == target:
            return i
        
     for j in range(len_):
        if nums[j] > target:
            return j 
        
     return len_     
        